Create ui-c8y-1021-22-0-allows-to-add-a-tab-to-the-alarm-detail-view.md (#2752)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-0-allows-to-add-a-tab-to-the-alarm-detail-view.md

---
date: '2024-12-19'

> title: Added a hook service to allow extension of alarm details view with custom tabs

Author: The Claude AI suggested to use the following title:
```suggestion
title: Extend alarm details with custom tabs
```

> title: allows to add a tab to the alarm detail view

Reviewer: ```suggestion
title: The alarm detail view has been extended with the possibility to add custom tabs.
```

> title: The alarm detail view has been extended with the possibility to add custom tabs.

Reviewer 2: ```suggestion
title: Enhanced alarm details view with the option to add custom tabs
```

ticket: MTM-59811
version: 1021.22.0
---

> allows to add a tab to the alarm detail view

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs in addition to the already existing "Info" and "History" tabs. This allows developers to display any kind of additional information in the context of an alarm. Existing alarm detail pages will not be affected by this change but it is now possible to enhance them with additional, use case specific tabs.
```

> allows to add a tab to the alarm detail view

Reviewer: ```suggestion
In some cases, it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs. This allows developers to display any kind of additional information in the context of an alarm. Existing alarm detail pages will not be affected by this change but it is now possible to enhance them with additional, use case specific tabs.
```

> In some cases, it may be required to show additional, custom information related to alarms. With this change, it is now possible to extend the alarm details view with custom tabs. This allows developers to display any kind of additional information in the context of an alarm. Existing alarm detail pages will not be affected by this change but it is now possible to enhance them with additional, use case specific tabs.

Reviewer 2: ```suggestion
It is now possible to extend the alarm details view with custom tabs. This allows developers to display any kind of additional information in the context of an alarm. Existing alarm details remain unchanged but can now be enhanced with additional custom tabs.
```
